source/VMelnalksnis.PaperlessDotNet/Documents/DocumentClient.cs

```diff
@@ -109,6 +110,23 @@ public async IAsyncEnumerable<Document<TFields>> GetAll<TFields>(int pageSize, [
 		return await GetCore<Document<TFields>>(id, cancellationToken).ConfigureAwait(false);
 	}
 
+
+	/// <inheritdoc />
+	public async Task<DocumentContent> Download(int id, bool original, CancellationToken cancellationToken = default)
+	{
+		var response = await _httpClient.GetAsync(original ? Routes.Documents.DownloadUri(id).ToString() :
+															 Routes.Documents.DownloadOriginalUri(id).ToString());
+
+		await response.EnsureSuccessStatusCodeAsync().ConfigureAwait(false);
+
+		var headers = response.Content.Headers;
+
+		return new DocumentContent(
+			content: await response.Content.ReadAsStreamAsync().ConfigureAwait(false),
```

Comment:
Should pass `cancellationToken`
```suggestion
			content: await response.Content.ReadAsStreamAsync(cancellationToken).ConfigureAwait(false),
```

Reply:
Looks like that overload is not available in .NET standard. In that case you'll have add target framework specific calls:
```csharp
#if NET6_0_OR_GREATER
await response.Content.ReadAsStreamAsync(cancellationToken).ConfigureAwait(false),
#else
await response.Content.ReadAsStreamAsync().ConfigureAwait(false),
#endif
```